src/Explore/DocumentItem.js

```diff
@@ -22,81 +31,144 @@ function DocumentItem(props) {
+            </Flex>
+            <ScoringIndicator score={score} />
+          </Flex>
+          <Heading
             as={Link}
             href={doiLink}
+            data-title
             target="_blank"
-            onMouseEnter={() => setIsHovered(true)}
-            onMouseLeave={() => setIsHovered(false)}
+            onClick={(evt) => {
+              evt.stopPropagation()
+            }}
             sx={{
-              color: isHovered ? 'text' : 'primary',
-              fontSize: 1,
-              ':hover': {
-                textDecoration: 'none',
-              },
+              display: 'block',
+              textDecoration: 'none',
+              ...(!isDetailed && {
+                whiteSpace: 'nowrap',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+              }),
             }}
           >
-            <FiExternalLink />
-            {doi}
-          </Box>
-          <ScoringIndicator score={score} />
-        </Flex>
-        <Heading
-          onMouseEnter={() => setIsHovered(true)}
-          onMouseLeave={() => setIsHovered(false)}
-          as={Link}
-          href={doiLink}
-          target="_blank"
-          sx={{
-            display: 'block',
-            whiteSpace: isDetailed ? 'wrap' : 'nowrap',
-            overflow: isDetailed ? 'visible' : 'hidden',
-            textOverflow: isDetailed ? 'none' : 'ellipsis',
-            textDecoration: isHovered ? 'underline' : 'none',
-          }}
-        >
-          {title}
-        </Heading>
-        <Box
-          onClick={() => isDetailed || setIsDetailed(true)}
-          sx={{
-            mx: -3,
-            mb: -3,
-            pb: 3,
-            px: 3,
-            pt: 1,
-            ':hover': {
-              bg: isDetailed || 'foreground',
-              cursor: isDetailed || 'pointer',
-            },
-          }}
-        >
-          {isDetailed ? <Detailed {...document} /> : <Simple {...document} />}
+            {title}
+          </Heading>
           <Box
-            onClick={() => setIsDetailed(false)}
-            sx={{
-              cursor: 'pointer',
-              textAlign: 'center',
-              fontSize: 4,
-              display: isDetailed ? 'block' : 'none',
-              mx: -3,
-              mb: -3,
-              ':hover': {
-                bg: 'foreground',
-              },
-            }}
+            sx={
+              !isDetailed && {
+                display: '-webkit-box',
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                '-webkit-line-clamp': '2',
+                '-webkit-box-orient': 'vertical',
+              }
+            }
           >
-            <FiChevronUp />
+            {summary}
           </Box>
         </Box>
+        {!isDetailed && (
+          <Flex
+            sx={{
+              flexDirection: ['column', 'row'],
+              alignItems: ['flex-start', 'center'],
+              fontSize: [0, 0, 1],
```

Comment:
I don't understand the point of making this smaller than regular text. The date and origin seem (to me) to have as much information value as the title and description.

Reply:
Alright, I'll bump it back up. :+1:

Reply:
Ah you did already, thanks :grin: